ui/src/message_popup/message_popup_page.jsx

```diff
@@ -67,10 +70,15 @@ export default React.createClass({
   getInitialState: function() {
     const isSolutionMode = _.has(this.props.query, 'solution');
     const helpType = isSolutionMode ? 'none' : 'feedback';
-    const shouldShowStudentCards = isSolutionMode ? false : _.has(this.props.query, 'cards');
+    //I changed the bottom to true since the feedback listed it as a bug but I thought that not
+    //showing the student cards in solution mode was the initial intention since the users would
+    //have known each of the students by then..?
```

Comment:
Ah yeah this is good clarification!

In general, the intent for solution mode was to not have the student cards available as support.  Candidates should have internalized these students and we'll expect them to demonstrate proficiency without that support.

But in playtests or other settings, we might have testers who don't already know these students so it's useful to have the option to provide the student cards in solution mode.  Last week, that was the original intention since everyone wouldn't know the students cold (and likely will be the way we'd do this in coming playtests as well).

So here I think we want the default to be `true` in practice mode, and `false` in solution mode, but let the query string override that in solution mode.